### PR TITLE
fix: tune Besu options to work with tx_spammer

### PR DESCRIPTION
**Pull request by fab-10**
At startup Besu does not return a good gas price estimation (WIP to fix this) and the default number of accepted connection is not enough to sustain the tx_spammer, so this PR tune some options to allow to use the tx_spammer wth Besu.

### CODE REVIEW

**Comment by fab-10** [comment]
@pk910 could you please review?

File: .github/tests/besu-all.yaml [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
     cl_type: nimbus
   - el_type: besu
     cl_type: lighthouse
   - el_type: besu
     cl_type: lodestar
-additional_services: []
+  - el_type: besu
+    cl_type: grandine
+additional_services:
+  - tx_spammer
```
Comment (barnabasbusa) [inline .github/tests/besu-all.yaml:15]:
```suggestion
additional_services: []
```

These tests are ran overnight. There is no need to test tx_spammer here. You can test that locally.

**Review by barnabasbusa — APPROVED** [review]

**Comment by pk910** [comment]
Yea, our testing workflow is currently broken for branches that are not part of this repository.
Right now it tries to find branch `tune-besu-for-tx-spammer` within this repository, but it's part of your fork, so it can't find it.
We'll get that fixed :)

**Comment by pk910** [comment]
PR workflow issue with assertoor got fixed with #616 
Sorry for the delay ;)

**Pull request merged**